README.md

```diff
@@ -0,0 +1,20 @@
+# secret_capture
+## Build a Screen recorder using python
+
+
+## Full tutorial (video)
+[![Watch the video-- Build your own screen](https://i.ytimg.com/an_webp/1J8dQA6gN7k/mqdefault_6s.webp?du=3000&sqp=CJ2sw4QG&rs=AOn4CLAgz0Xro-XZWrk3gI99epHO3te9Dg)](https://www.youtube.com/watch?v=1J8dQA6gN7k "Build your own screen recorder")
+
+
+## Installation
+### For windows users
+(run those in command prompt/cmd/terminal)
+
+Download requirements.txt file
+``` 
+pip install -r requirements.txt
+```
+
+
+#### Issues
+If you encounter any problems feel free to open a new issue. Before that check other closed issues and check if your issue matches with any older issues.
```

Comment:
Make the `open a new issue` hyperlink. 
If you incounter any issue, feel free to [open a new issue.](https://github.com/ProgrammingHero1/secret_capture/issues/new) Before that........